dev/com.ibm.ws.microprofile.faulttolerance.cdi/resources/com/ibm/ws/microprofile/faulttolerance/cdi/resources/FaultToleranceCDI.nlsprops

```diff
@@ -134,6 +134,11 @@ fallback.method.not.found.CWMFT5021E=CWMFT5021E: The fallback method {0} with pa
 fallback.method.not.found.CWMFT5021E.explanation=A fallback method was declared but it does not exist in the declaring class, its superclasses, or the interfaces it implements.
 fallback.method.not.found.CWMFT5021E.useraction=Either remove the fallback annotation or add the missing method. If the fallback method exists but is still not being found, check that the parameter types and return type of the two methods match and that the fallback method is visible to the class which declares the annotated method.
 
+# Conflict with Concurrency's @Asynchronous
+anno.conflict.CWMFT5022E=CWMFT5022E: MicroProfile Fault Tolerance annotations cannot be used in combination with {0}.
```

Comment:
MicroProfile Fault Tolerance annotations cannot be used in combination with the {0} annotation.

Reply:
@dmuelle thanks for reviewing!  Both of your suggestions have been made exactly as suggested under https://github.com/OpenLiberty/open-liberty/pull/20162/commits/a9694cf08abd903cb74b28060dc12cac71bd4f88
Please approve your review once you confirm.